RevenueCatUI/Templates/V2/Components/Video/VideoComponentViewModel.swift

```diff
@@ -93,7 +91,18 @@ class VideoComponentViewModel {
 
         apply(style)
     }
+}
 
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+extension VideoComponentViewModel: Hashable {
+    func hash(into hasher: inout Hasher) {
+        hasher.combine(component)
+        hasher.combine(imageSource)
```

Comment:
Nit: I think `imageSource` is not needed here since it's actually the `component.fallbackSource` (so it's already considered in the `component`'s hash. 
That said, we might want to leave it here counting for the hash in case we change its implementation in the future? (i.e. imageSource coming from somewhere else?)